carsons/carsons.py

```diff
@@ -183,3 +181,75 @@ def calculate_distance(positionᵢ, positionⱼ):
     def get_h(self, i):
         _, yᵢ = self.phase_positions[i]
         return yᵢ
+
+    @property
+    def impedance(self):
```

Comment:
i think this new property should be tested. perhaps also deprecate the `convert_geometric_model` function

Reply:
it is called on the concentric neutral. its just added here because it doesn't depend on anything that `Carsons` doesn't have.

Reply:
exposed `impedance` as a stand-alone function in 723ff87